### ISSUE TITLE
Simple test of gitbreaker in vanilla JS (no build tool)

### ISSUE THREAD

**Issue by deguich** [jdalrymple/gitbeaker#3237]
**Description**

- Gitbeaker version: 38.1.0

I am trying to use gitbreaker in browser and vanilla JS : 

```html
<!doctype html>
<html lang="en">
  <head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>gitbreaker test</title>
  </head>
  <body>
    <script type="module">
	  import { Gitlab } from 'https://cdn.skypack.dev/@gitbeaker/rest';  
    </script>
  </body>
</html>
```

Error in JS console : 

Uncaught TypeError: Failed to resolve module specifier "@gitbeaker/core". Relative references must start with either "/", "./", or "../".

What I am doing wrong ?

Thx in advance

**Comment by jdalrymple** [comment]
Yes, there is definitely a problem. Ill look into it and try to include skypack into my tests to avoid this in the future.

**Comment by deguich** [comment]
It works with esm.sh url.

**Comment by deguich** [comment]
Just for information, documentation show the broken library link https://cdn.skypack.dev/@gitbeaker/rest : 
https://www.npmjs.com/package/@gitbeaker/rest

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`38.2.0`](https://github.com/jdalrymple/gitbeaker/releases/tag/38.2.0) :rocket:
